fahrschein-example/build.gradle

```diff
@@ -9,16 +9,16 @@ dependencies {
     implementation project(':fahrschein-http-spring')
     implementation project(':fahrschein-inmemory')
     implementation project(':fahrschein-typeresolver')
-    implementation "com.squareup.okhttp3:okhttp:${property('okHttpV')}"
-    implementation "org.apache.httpcomponents:httpclient:${property("apacheHttpV")}"
-    implementation "org.springframework:spring-web:${property('springV')}"
+    implementation "com.squareup.okhttp3:okhttp:${property('okhttp.version')}"
+    implementation "org.apache.httpcomponents:httpclient:${property("apachehttp.version")}"
+    implementation "org.springframework:spring-web:${property('spring.version')}"
     implementation 'org.zalando:jackson-datatype-money:0.6.0'
```

Comment:
this is sooooo outdated!

Reply:
You mean the last update of this library was 2015?

Reply:
The version we're pulling in here (0.6.0) is from 2015.  Last release (1.3.0) is from Nov 2021.

Reply:
Is there a blocker why we cannot update it to the latest one?

Reply:
no, but I wouldn't want to upgrade dependencies as part of this PR.